### DISCUSSION TITLE
What animal is the mascot of Kellogg's cereal, Frosted Flakes?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#632] [Galaxy Trivia] [answered]
**Bonus:** What is the name of the mascot of Froot Loops?

**Comment by peterbaccaro** [comment, marked as answer]
The animal mascot of Kellogg's cereal, Frosted Flakes, is a tiger named **Tony the Tiger**.  Tony has been the brand's mascot since 1952 and is known for his catchphrase "They're Grrreat!"

The name of the mascot of Froot Loops is **Toucan Sam**. Toucan Sam is a colourful toucan bird with a white body, blue beak, and colourful feathers on his head. He has been the mascot of Froot Loops since the 1960s and is known for his catchphrase "Follow your nose! It always knows!"

**Reply by ParanoidUser** [reply]
"You nailed it, you got it right! Thank you for your answer, it's out of sight!" 😎
